www/app/docs/user-authentication/[[...slug]]/enterprise-login/okta.mdx

```diff
@@ -33,9 +35,20 @@ cd nile-okta-nextjs
 
 and install dependencies with `yarn install` or `npm install`.
 
-Rename `.env.local.example` to `.env.local`, and update it with your workspace and database name.
+## 2. Configure your application.
+
+- Rename `.env.local.example` to `.env.local` in the project's root directory and open it.
+- Enter your database credentials (`NILEDB_USER` and `NILEDB_PASSWORD`). Use existing credentials, or generate new ones through the Nile Console:
+
+1. Visit the [Nile Console](https://console.thenile.dev), select your database, and navigate to **Settings** > **Credentials**.
+2. Click **Generate credentials**, then **Generate** and **Copy credentials to clipboard**.
+3. The copied credentials include two UUIDs: the first is the username and the second is the password, separated by a colon (":"). Paste the username into the `NILEDB_USER` value in `.env.local`, and the password into the `NILEDB_PASSWORD` value.
+
+These credentials authenticate your app with the Nile API.
+
+- In the same `.env.local` file, enter the `NEXT_PUBLIC_NILEDB_ID`. You can find this in the [Nile Console](https://console.thenile.dev) under **Settings** > **General**.
```

Comment:
Note: a couple of these examples rely on the database id, and assumes that we'll add it back to NAD.

Reply:
You can't assume this. We agreed on the NAD UI. 

We need to fix the example.